tests/integration-tests/commit_deployments_test.go

```diff
@@ -0,0 +1,161 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com*/
+
+package integration_tests
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"math/rand"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	json "google.golang.org/protobuf/encoding/protojson"
+
+	"github.com/freiheit-com/kuberpult/pkg/api/v1"
+)
+
+const (
+	development = "development2"
+	staging     = "staging"
+)
+
+func TestCommitDeployments(t *testing.T) {
+	randNum := fmt.Sprintf("%d", rand.Intn(10000))
```

Comment:
Why do we need to randomize the test? 
If this is really necessary, we need to document why.
But I would much prefer to remove any randomness.

Reply:
The test assumes that the apps do not exist, and builds on that assumption. If the test was to run multiple times, that assumption would be incorrect and the test would fail.

We might not have to the case where we want to run the integration tests more than once, so I will remove this randomization.